src/io/reader.cpp

```diff
@@ -6,18 +6,17 @@
 
 namespace copc
 {
-Reader::Reader(std::istream &in_stream) : in_stream_(in_stream)
+
+Reader::Reader(std::istream *in_stream) : in_stream_(in_stream) { InitReader(); }
```

Comment:
We could move this to the header if you want since the FileReader one is in the header